sdk/keyvault/Azure.Security.KeyVault.Secrets/api/Azure.Security.KeyVault.Secrets.netstandard2.0.cs

```diff
@@ -65,31 +65,56 @@ public partial class SecretClient
         protected SecretClient() { }
         public SecretClient(System.Uri vaultUri, Azure.Core.TokenCredential credential) { }
         public SecretClient(System.Uri vaultUri, Azure.Core.TokenCredential credential, Azure.Security.KeyVault.Secrets.SecretClientOptions options) { }
+        public virtual Azure.Core.Pipeline.HttpPipeline Pipeline { get { throw null; } }
         public virtual System.Uri VaultUri { get { throw null; } }
+        public virtual Azure.Response BackupSecret(string secretName, Azure.RequestContext context = null) { throw null; }
```

Comment:
To avoid conflicts, we were planning to make the RequestContext parameter required.